### ISSUE TITLE
Station closes when I close a weird blank "download" window that appears using Slack or leave a call/huddle.

### ISSUE THREAD

**Issue by tuqueque** [getstation/desktop-app#331]
Hello, I've been using Ferdium for the last year or so and recently I discovered Station, I have to say Station looks better and the initial setup is easier, but Ferdium gives me more control and options for each service and has a better handling of notifications... I hope Station brings those improvement and finer control as well... Anyway, that's not the reason for this bug report.

The problem I've been having for the last couple of days with Station (to the point that it would be a deal breaker for me if it doesn't get fixed soon) is that when I download a file from Slack, a new, blank window opens (I don't know why), when I close that window, a second later Station also closes... The same happens if I join a call (or "huddle", as called in Slack)... If I "maximize" the call to a separate winwow, when I finish the call and press the "Leave" button, Station closes a second later.

It seems like Station doesn't handle too well new windows.

It happens with the version installed from the AUR repository in Manjaro Linux and the .appimage here in Github.

As mentioned, my system is Manjaro Linux.

Thank you in advance.

**Comment by viktor44** [comment]
What is the version of Station?

**Comment by tuqueque** [comment]
Oh, I forgot to mention, the latest one: 2.7.3

**Comment by viktor44** [comment]
Thank you for the report about blank download window in Slack. I've fixed it.

I don't have Manjaro Linux but I've checked Debian and CentOS.
Station doesn't end in described situation. 
Your description looks like application crash. Maybe Electron is not fully compatible with your OS...

**Comment by tuqueque** [comment]
Thank you very much for the fix. I think this will pretty much solve the problem. If there's no blank window to close, I don't have to worry about Station being closed in consequence.

Also... Yesterday I downloaded a file in Slack and I didn't want to close the blank window to avoid having to re-open Station, so I left that blank window there for 3-4 hours until I finished using Slack. When I finally closed that window, Slack didn't close in consequence... So maybe some process is running under the hood in that window (besides the actual download of the file) for some time and I needed to let it finish before closing it... In any case, you say that window will no longer appear, so again, thank you very much for solving the problem so quickly!

I'll wait for the new version of Station to test it out.
